jaxrs/pom.xml

```diff
@@ -38,8 +38,8 @@
 
     <dependency>
       <groupId>javax.ws.rs</groupId>
-      <artifactId>jsr311-api</artifactId>
-      <version>1.1.1</version>
+      <artifactId>javax.ws.rs-api</artifactId>
```

Comment:
Hmm. Dangerous thing to do in a minor, changing a dependency like this

Reply:
how? jax-rs-2.1 is fully backwards compatible down to 1.1.

Reply:
I think one way to address this, is to keep the original dependency and add the new one as provided.

For those using jax-rs 2, they would need to explicitly declare the dependency. And for the ones already using jax-rs 1 no need for changes.

Reply:
@spencergibb how is this a minor btw? I'm actually considering jax-rs-2.x breakage pretty much a major considering there's no applicable workaround and the spec has been around for half a decade by now. In consideration of 2.x being backwards compatible thus applicable as drop-in substitute, would make  jsr-311 legacy support pretty much a minor instead.